en/includes/references/conditional-auth/api-reference.md

```diff
@@ -723,6 +724,32 @@ This function returns the local user associated with the federate username given
       </tbody>
     </table>
 
+### Get masked value
+
+`getMaskedValue(value)`
+
+This utility function returns a masked value for the given input value. This function can be used to mask sensitive content in adaptive authentication script logs by developers.
+
+- **Parameters**
+
+    <table>
+      <tbody>
+        <tr>
+          <td><code>value</code></td>
+          <td>Value needs to be masked.</td>
+        </tr>
+      </tbody>
+    </table>
+
+- **Example**
+
+  For debugging purposes, if there is a need to add some logs including sensitive content (ex: PII) then it can be masked using the above method as follows.
```

Comment:
Since we have used the word `function` to identify these methods through out the doc, shall we use that wording instead of `method`, to be consistent?

Reply:
Addressed with https://github.com/wso2/docs-is/pull/4664/commits/2fd65e972210c46c6302574ff5035c6cfda5e576